nd4j/RaspberryPi.md

```diff
@@ -0,0 +1,46 @@
+Please follow following instructions to build nd4j for raspberry PI:
+
+1. download cross compilation tools for Raspberry PI
+
+    ```
+    $ apt-get/yum install git rsync cmake
+    $ mkdir $HOME/raspberrypi
+    $ cd $HOME/raspberrypi
+    $ git clone git://github.com/raspberrypi/tools.git
+    $ export PATH=$PATH:$HOME/raspberrypi/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin
+    $ mkdir $HOME/raspberrypi/rootfs
+    (Replace the ip address in the following command with the IP of a raspberry pi device)
+    $ rsync -rl --delete-after --safe-links pi@(192.168.1.PI):/{lib,usr} $HOME/raspberrypi/rootfs
+    $ cat > $HOME/raspberrypi/pi.cmake <<END
+    SET(CMAKE_SYSTEM_NAME Linux)
+    SET(CMAKE_SYSTEM_VERSION 1)
+    SET(CMAKE_C_COMPILER $ENV{HOME}/raspberrypi/tools/arm-bcm2708/arm-rpi-4.9.3-linux-gnueabihf/bin/arm-linux-gnueabihf-gcc)
```

Comment:
Let's use a variable for the path to the tools.

Reply:
All 3 like for Android here:
https://github.com/deeplearning4j/deeplearning4j/blob/master/libnd4j/cmake/android-arm.cmake
Or just 1 variable that's used for all 3. It doesn't really matter, as long as it's configurable so we can include it in a script file like this:
https://github.com/deeplearning4j/deeplearning4j/blob/master/libnd4j/buildnativeoperations.sh#L150

Reply:
Ahh, got it.  That makes a lot of sense for how to organize things, so I'll check out the repo locally and properly edit this branch.

Reply:
Modified and modeled after Android.